### DISCUSSION TITLE
Changing font sizes - how?

### DISCUSSION THREAD

**Discussion by LocalJoost** [mmistakes/minimal-mistakes#2629] [Q&A] [answered]
I have spent literally _hours_ on this, and I simply have to admit now this is beyond me. I have created my blog using this awesome theme and have been happily customizing it using my rather rusty web dev skills. But I really can't find how I simply can change font sizes.

For instance: if you look at this, I find the text in the middle pane (containing the actual blog text) a bit big. So I want to change it. In https://localjoost.github.io/migrating-for-mrkt2-nuget-for-unity/ this for instance the text
"Recently I was setting up a new HoloLens 2 project. I usually take the following route" 

Things I tried, with no effect:

- Changing the 3 sizes in the .html tag in reset.scss
- Changing the type scales in variables.scss. This had some effect, but I could only make them bigger. As in WAY bigger
- Hacking values directly in main.scss.

And a lot of other things I can't even remember. Apparently there is a smart mechanism that resizes the text when your decrease the windows size - I assumed that was the media query in reset.scss but apparently that is not the case.

Can you _please_ give me some pointers? Thanks!

**Comment by mmistakes** [comment, marked as answer]
Have a look at these duplicate issues for pointers. I've covered this one in detail many many times before.

Ref: #1219, #1352, #1184, #2290, #2435

**Comment by LocalJoost** [comment]
Okay, I stand corrected. I have tried this before and it did not seem to work. I must have done something wrong, apparently.

Joost van Schaik

Senior Software Architect Mixed Reality
Windows Development MVP<https://mvp.microsoft.com/en-us/PublicProfile/4034925> & Windows Insider MVP<https://insider.windows.com/en-us/mvps/joost-van-schaik>

[cid:image010.png@01D66290.EFE73CD0]

LinkedIn<https://linkedin.com/in/joostvanschaik>  Blog<https://dotnetbyexample.blogspot.com>  Twitter<https://twitter.com/localjoost>

Sent from mail for Windows 10

"The world as I envision it in my head is such a more interesting place"
